orca-clouddriver/src/test/java/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/WaitForManifestStableTaskTest.java

```diff
@@ -56,7 +61,7 @@ void terminalWhenFailedStable() {
         .thenReturn(manifestBuilder().stable(true).failed(true).build());
 
     TaskResult result = task.execute(myStage);
-    AssertionsForClassTypes.assertThat(result.getStatus()).isEqualTo(ExecutionStatus.TERMINAL);
+    assertThat((CharSequence) result.getStatus()).isEqualTo(ExecutionStatus.TERMINAL);
```

Comment:
Hmmm...It seems AssertionsForClassTypes was there for a reason -- to avoid this cast.  Would you mind restoring it?  If you're over it, I understand and we can merge this as it is.